inference-engine/src/transformations/src/transformations/common_optimizations/remove_concat_zero_dim_input.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/remove_concat_zero_dim_input.hpp"
+#include "transformations/utils/utils.hpp"
+
+#include <memory>
+#include <vector>
+#include <algorithm>
+
+#include <openvino/opsets/opset8.hpp>
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include <ngraph/rt_info.hpp>
+#include "itt.hpp"
+
+
+NGRAPH_RTTI_DEFINITION(ov::pass::RemoveConcatZeroDimInput, "RemoveConcatZeroDimInput", 0);
+
+ov::pass::RemoveConcatZeroDimInput::RemoveConcatZeroDimInput() {
```

Comment:
Let's include it inside NopElimination transformation

Reply:
done